tests/integrate/time.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+total_time=0
+
+# 遍历文件夹下的所有子文件夹
```

Comment:
change Chinese to English

Reply:
This file was mistakenly included and has now been removed.